DOC-1642: Release notes for 6.0.2 (#2415)
tinymce/tinymce-docs: modules/ROOT/pages/6.0.2-release-notes.adoc

= TinyMCE 6.0.2
:navtitle: TinyMCE 6.0.2
:description: Release notes for TinyMCE 6.0.2
:keywords: releasenotes, new, changes, bugfixes

include::partial$misc/admon-releasenotes-for-stable.adoc[]

[[overview]]
== Overview

{productname} 6.0.2 was released for {enterpriseversion} and {cloudname} on Wednesday, May 4^th^, 2022. It includes {productname} 6.0.2 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.0.2, including:

* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
* xref:general-bug-fixes[General bug fixes]
* xref:known-issues[Known issues]

[[accompanying-premium-plugin-changes]]
== Accompanying Premium Plugin changes

The following premium plugin updates were released alongside {productname} 6.0.2.

=== Spell Checker Pro 3.0.2

The {productname} 6.0.2 release includes an accompanying release of the **Spell Checker Pro** premium plugin.

> **Spell Checker Pro** 3.0.2 fixes the page focus incorrectly changing when spellchecking is triggered on multiple editors after scrolling.

Author: ```suggestion
In the previous release, **Spell Checker Pro** would change page focus incorrectly when spellchecking was triggered on multiple editors after a `scroll` event.

**Spell Checker Pro** 3.0.2, which accompanies this release, corrects this so that the page focus no longer moves in this circumstance.
```

> 

Reviewer: ```suggestion
```
There shouldn't be a blank line here.

= TinyMCE 6.0.2
:navtitle: TinyMCE 6.0.2
:description: Release notes for TinyMCE 6.0.2
:keywords: releasenotes, new, changes, bugfixes

include::partial$misc/admon-releasenotes-for-stable.adoc[]

[[overview]]
== Overview

{productname} 6.0.2 was released for {enterpriseversion} and {cloudname} on Wednesday, May 4^th^, 2022. It includes {productname} 6.0.2 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.0.2, including:

* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
* xref:general-bug-fixes[General bug fixes]
* xref:known-issues[Known issues]

[[accompanying-premium-plugin-changes]]
== Accompanying Premium Plugin changes

The following premium plugin updates were released alongside {productname} 6.0.2.

=== Spell Checker Pro 3.0.2

The {productname} 6.0.2 release includes an accompanying release of the **Spell Checker Pro** premium plugin.

> **Spell Checker Pro** 3.0.2 provides the following bug fixes:
> 
> * Editor focus was incorrectly changed when spellchecking multiple editors after scrolling.

Reviewer: The old rule Tyler had was we shouldn't have a list with just one item, so maybe something like this instead (just using what I used in 5.10.4)?
```suggestion
**Spell Checker Pro** 3.0.2 fixes the page focus incorrectly changing when spellchecking is triggered on multiple editors after scrolling.
```

The {productname} 6.0.2 release includes an accompanying release of the **Premium Skins and Icon Packs**.

=== Premium Skins and Icon Packs - Bug fixes

The **Premium Skins and Icon Packs** release includes the following bug fixes:

* Added background and additional spacing for the text labeled buttons in the toolbar to improve visual clarity.
* Toolbar split buttons with text used an incorrect width on touch devices.

For information on using premium skins and icon packs, see: xref:premium-skins-and-icons.adoc[Premium Skins and Icon Packs].

[[general-bug-fixes]]
== General bug fixes

{productname} 6.0.2 provides fixes for the following bugs:

* Some media elements wouldn't update when changing the source URL.
* Inline toolbars flickered when switching between editors.
* Multiple inline toolbars were shown if focused too quickly.
* Added background and additional spacing for the text labeled buttons in the toolbar to improve visual clarity.
* Toolbar split buttons with text used an incorrect width on touch devices.

[[known-issues]]
== Known issues

This section describes issues that users of {productname} 6.0.2 may encounter and possible workarounds for these issues.

> There are no known issues affecting {productname} 6.0.2.

Reviewer: This isn't accurate, as there are definitely known issues, there's just no new ones. We normally omit that section in that case, but at the same time this seems like it could be useful?

If we're going to keep this, then maybe something like this:
```suggestion
There are no new known issues affecting {productname} 6.0.2.
```

Author: I mainly kept the section so we could have something to link to for the left sidebar nav. You are right that saying 'new' is more accurate so will add that. Thanks
